tests/operator_test.go

```diff
@@ -376,6 +408,43 @@ var _ = Describe("Operator", func() {
 
 		})
 
+		It("should be able to update kubevirt install when operator updates if no custom image tag is set", func() {
```

Comment:
nice how easy it is to add such a new test, using (almost) all of the existing funcs